runtime/gc_modron_startup/mminit.cpp

```diff
@@ -3228,6 +3231,9 @@ gcReinitializeDefaultsForRestore(J9VMThread* vmThread)
 			 * always pass false to computeDefaultMaxHeapForJava().
 			 */
 			candidateSoftMx = extensions->computeDefaultMaxHeapForJava(false);
+			if (extensions->testContainerMemLimit) {
+				j9tty_printf(PORTLIB, "Current heapAlignment value is: %llu\n", extensions->heapAlignment);
```

Comment:
Better, VGC report ("container memory limit set") should be adjusted, but that's OMR code and would not be able to see this J9 specific flag. I'm fine with not reporting anything (so, remove this print).

Reply:
We can do VGC report.

Current test is based on this print, since we need to know the heapAlignment value for the roundToFloor function.